### DISCUSSION TITLE
Forking libsoundio

### DISCUSSION THREAD

**Discussion by Kreijstal** [LMMS/lmms#7617] [Project Management]
while trying to package libsoundio for msys2 I've noticed there has been little to no engagement since a long time ago, since you depend on it, maybe it isn't a bad idea if you fork it, accept contributions, bug reports, etc?
https://github.com/andrewrk/libsoundio/issues/301

**Comment by Rossmaxx** [comment]
We already have very few devs to maintain the LMMS core project and i believe the project shouldn't take up any more side projects, though forking libsoundio is a reasonable bet, but the LMMS team is not in a position to take it up. But if someone from us does decide to take it up, I'm ok with it.

**Comment by tresf** [comment]
> while trying to package libsoundio for msys2 I've noticed there has been little to no engagement since a long time ago, since you depend on it, maybe it isn't a bad idea if you fork it, accept contributions, bug reports, etc? [andrewrk/libsoundio#301](https://github.com/andrewrk/libsoundio/issues/301)

Yeah, he's the guy that created [Zig](https://ziglang.org/).  He's probably busy on other things these days.

I'm not sure how valuable the library is to LMMS so any talks of forking should be proceeded with a conversation about whether or not we actually want/need it.
